File: app/src/main/java/com/quiz/mallzellij_show_android/model/InventoryRequest.java
package com.quiz.mallzellij_show_android.model;

import java.math.BigDecimal;

public class InventoryRequest {
    private String ynum0;
    private String ydepot0;
    private String yequipe0;
    private String yzone0;
    private String yitmref0;
    private BigDecimal yqtyplt0;
    private BigDecimal yqtycrt0;
    private BigDecimal yqtymtr0;
    private String creusr0;

    public InventoryRequest(String ynum0, String ydepot0, String yequipe0, String yzone0,
                            String yitmref0, BigDecimal yqtyplt0, BigDecimal yqtycrt0,
                            BigDecimal yqtymtr0, String creusr0) {
        this.ynum0 = ynum0;
        this.ydepot0 = ydepot0;
        this.yequipe0 = yequipe0;
        this.yzone0 = yzone0;
        this.yitmref0 = yitmref0;
        this.yqtyplt0 = yqtyplt0;
        this.yqtycrt0 = yqtycrt0;
        this.yqtymtr0 = yqtymtr0;
        this.creusr0 = creusr0;
    }
}

File: app_flutter/lib/models/inventory_request.dart
class InventoryRequest {
  final String ynum0;
  final String ydepot0;
  final String yequipe0;
  final String yzone0;
  final String yitmref0;
  final double yqtyplt0;
  final double yqtycrt0;
  final double yqtymtr0;
  final String creusr0;

  InventoryRequest({
    required this.ynum0,
    required this.ydepot0,
    required this.yequipe0,
    required this.yzone0,
    required this.yitmref0,
    required this.yqtyplt0,
    required this.yqtycrt0,
    required this.yqtymtr0,
    required this.creusr0,
  });

  Map<String, dynamic> toJson() => {
    'ynum0': ynum0,
    'ydepot0': ydepot0,
    'yequipe0': yequipe0,
    'yzone0': yzone0,
    'yitmref0': yitmref0,
    'yqtyplt0': yqtyplt0,
    'yqtycrt0': yqtycrt0,
    'yqtymtr0': yqtymtr0,
    'creusr0': creusr0,
  };
}
